cypress/integration/data-import/match-on-holdings-856-u.spec.js

```diff
@@ -97,12 +89,11 @@ describe('ui-data-import: Match on Holdings 856 $u', () => {
   });
 
   after(() => {
-    HoldingsRecordView.getId().then((id) => {
-      cy.deleteHoldingRecordViaApi(id);
-    });
-    InventoryInstance.getId().then(id => {
-      InventoryInstance.deleteInstanceViaApi(id);
-    });
+    cy.getInstance({ limit: 1, expandAll: true, query: `"title"=="${instanceTitle}"` })
```

Comment:
Are you sure that this instance with instanceTitle always exists?

Reply:
yes, in this instance is in a .mrc file and create always when the file is uploaded